### ISSUE TITLE
It sais that the mod is "deprecated and could stil be broken" in the tunderstore and does not work

### ISSUE THREAD

**Issue by Maxace1** [xoxfaby/BetterUI#193]
It sais that the mod is "deprecated and could stil be broken" in the tunderstore and does not work

![image_2024-06-12_154338808](https://github.com/xoxfaby/BetterUI/assets/172511189/3c6e6e85-a372-46c7-ac86-31f13c11b659)
)

**Comment by krama** [comment]
> It sais that the mod is "deprecated and could stil be broken" in the tunderstore and does not work
> 
> ![image_2024-06-12_154338808](https://private-user-images.githubusercontent.com/172511189/338993039-3c6e6e85-a372-46c7-ac86-31f13c11b659.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5OTkxNjYsIm5iZiI6MTcxODk5ODg2NiwicGF0aCI6Ii8xNzI1MTExODkvMzM4OTkzMDM5LTNjNmU2ZTg1LWEzNzItNDZjNy1hYzg2LTMxZjEzYzExYjY1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMVQxOTQxMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTUxMWFjNGI5NTJhODVhNTkxYTIzYTNhZjU1ZWUwNzY5MWE0ODMyNTY4YjhiMjIyMjAzYmM4MDZlMzhkZWVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._sAGVpMcB2LJAtvdRFOWEV-9dBL4lVhK6eHUkPzS3es) )

Yes, the author decided to rage quit the community, and it's one thing if he had dropped a mod that unlocks everything or BetterAPI or any other mod. But when you have a library that affects almost 900 other works, it can only be called sabotage, unfortunately. On the other hand, we don't know the details, and personally, I'm not particularly interested in what happened. Yes, it got worse, but on the other hand, sooner or later, a similar mod or an alternative will be made.

**Comment by CallMeYuuuu** [comment]
"BetterUI
Do not try to fork this Repo to bring back old versions of the mod, they will be taken down.
A mod that makes the UI better by adding a nice little button to the main menu

All other features of BetterUI have been removed.

This change was made due to continued harrassment from a part of the RoR2 Modding community and Thunderstore's lack of action regarding these spite-fueled individuals.

If you want to follow my other stuff or contact me about this, feel free to join my discord at https://discord.faby.dev/ but please be aware that this decision is final."
